### DISCUSSION TITLE
Which Plugins do I need for your Templates?

### DISCUSSION THREAD

**Discussion by dredhorse** [BryanJenksCommunity/FAQ#149] [Q&A] [answered]
Hi,

I haven't found a list of required plugins in your ZIP file.

Do you have a list?

**Comment by tallguyjenks** [comment, marked as answer]
with that zipfile its a template vault, you just open it in obsidian and turn on community plugins and update and it should install everything used in that template vault :)

**Reply by dredhorse** [reply]
just as a note to all others who stumble upon this:

Keep in mind that .obsidian (the plugin and config folder) is normally not visible in MACOS. So moving the vault needs special care 😀

**Reply by tallguyjenks** [reply]
and to make it visible you can press ⌘+⇧+. | <kbd>⌘⇧.</kbd>

**Comment by krzakx** [comment]
Speaking about plugins.
![image](https://user-images.githubusercontent.com/18627339/206433239-5bb40d8a-b9c6-4a30-97e5-64cb6b7cb157.png)
Are those somehow tripled?
